packit_service/service/app.py

```diff
@@ -64,8 +66,10 @@ def get_flask_application():
     return app
 
 
-application = Proxy(get_flask_application)
+packit_as_a_service = Proxy(get_flask_application)
 
+# Make Prometheus Client serve the /metrics endpoint
+application = DispatcherMiddleware(packit_as_a_service, {"/metrics": prometheus_app()})
```

Comment:
I naively thought it's /api/metrics; had to look here to realize it's really just /metrics

Reply:
same here :D

Reply:
:see_no_evil: sorry guys, I'm not even sure if `/api/metrics` would have been possible, and my logic was that `/metrics` is not really part of the application API (but I'm aware that that's debatable).